app/auth-portal/src/content/tutorial/08-asset_authoring.md

```diff
@@ -0,0 +1,442 @@
+---
+title: Asset Authoring
+---
+
+## How to create an Asset in System Initiative
+
+Today, we are going to build a System Initiative asset that allows us to create and manage
+[AWS EBS Volumes](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ebs-volumes.html). We are going to create a 100GB EBS Volume in AWS Ohio (us-east-2a)
+that has 3000 iops, running on an SSD and is unencrypted.
+
+**Please take care to follow the guide precisely; there's a lot of rough edges. Specifically,
+ensure you stick with the function naming scheme and setting the appropriate properties of
+attached functions.**
+
+### Anatomy of an Asset
+
+An Asset is a collection of functions that come together to provide the capability to talk
+to an infrastructure provider, e.g. an AWS EBS Volume. An asset is made of the following
+types of functions:
+
+* Schema Variant Definition
+* Qualification
+* Action
+* Code Generation
+* Attribute
+* Validation
+
+The type of functions that an asset needs will depend on what the asset does. For example,
+if we want our asset to have a real world representation of our 'digital twin', then we
+need `action functions` to be able to _take action_ with the infrastructure provider. Or, you
+need need `attribute functions` if you want to manipulate data going
+to the asset or from the asset.
+
+### Building the Schema Variant Definition of AWS EBS Volume
+
+A Schema Variant Definition, is a function that allows us to create the structure of the
+asset schema. This definition is, usually, made up of:
+
+* Props
+* Input Sockets
+* Output Sockets
+
+There are other parts to the schema definition, but these are the most common. You can find the
+full schema definition in our [source code](https://github.com/systeminit/si/blob/main/bin/lang-js/src/asset_builder.ts).
+In System Initiative, we try and model out schema similar to how the AWS ecosystem would model
+concept. I usually use the [AWS CLI Documentation](https://docs.aws.amazon.com/cli/latest/reference/#cli-aws)
+to understand what the schema needs to include. It's important to note that as we build out assets, it's easy to
+extend the asset, so you don't need to cover all scenarios in the first creation of the asset.
+
+Looking at the AWS CLI documentation for [creating an EBS Volume](https://docs.aws.amazon.com/cli/latest/reference/ec2/create-volume.html)
+I can see that we will want to have the following properties:
+
+* availability-zone
+* encrypted | not-encrypted
+* iops
+* kms-key-id
+* size
+* volume-type
+* region
+
```

Comment:
If you put these in the exact order you'll address them in the code - it'll be easier to follow